website/static/_redirects

```diff
@@ -2,4 +2,4 @@
 /docs/tasks/running-pods/       /docs/tasks/scheduling/
 /docs/provisioner-crd/	        /docs/provisioner/
 /docs/faqs/	                    /docs/faq/
-/docs/*     	                /v0.13.2/:splat
+/docs/*     	                /v0.14.0-rc.0/:splat
```

Comment:
I don't think we want to redirect the latest version to 0.14

Reply:
Yup I was thinking that too. I'll change.